torch/csrc/cudnn/Conv.cpp

```diff
@@ -282,6 +409,11 @@ void findAlgorithm(
     return;
   }
 
+  if (deterministic) {
+    *algo = search::DEFAULT_ALGO;
```

Comment:
is there really only one deterministic algorithm in cuDNN?!

Reply:
that's what I saw in the cudnn documentation